src/components/layouts/pages/inspect/validationErrorMessages_en.json

```diff
@@ -3,7 +3,15 @@
     "1.3.1": {
       "1": {
 	"SUMMARY": "Is this meant to be a line or figure?",
-	"DESCRIPTION": "There are many repeated occurrences of the same character in this line. Beware that reader software for people with visual impairments will read these out one by one. This can be confusing and annoying for the reader. If you were trying to make a line, figure or form then insert a real line of figure. Don't use repeated underscores, dashes or other characters."
+	"DESCRIPTION": "There are many repeated occurrences of the same character in this line. Beware that reader software for people with visual impairments wil read these out one by one. This can be confusing and annoying for the reader. If you were trying to make a line, figure or form then insert a real line of figure. Don't use repeated underscores, dashes or other characters."
```

Comment:
wil instead of will

Reply:
Thanks for reviewing and for identifying this bug. I fixed it in `patch-19`.